client/src/app/containers/Game/slice.js

```diff
@@ -0,0 +1,172 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const initialState = {
+  currentGameID: '',
+  currentGame: {},
+  currentAction: {
+    numShuffles: 0,
+    selectedCards: [],
+    percCut: 0.5,
+    points: -1,
+  },
```

Comment:
These are all transient things which get sent along with player actions. Do they really need to be in the state?

Reply:
🤷  This is one place I'm looking for React client design advice/knowledge. Somehow the saga needs to pull the state out of the components, and this is the only way I know how to do it so far. I won't be surprised if there's a better way.

Reply:
I think what you want here is the `useState` hook. Then you can pass the state into your action creator from your component, and the saga will have it. Remember: `useState` is used to set up and manage component-level state (like form data, the value of a slider, etc.), whereas state in the redux store is application-level state (logged in user, current game, etc.).